### PR TITLE
schild-report.schild-report version 2.4.0.0

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25124)

### CODE REVIEW

File: manifests/s/schild-report/schild-report/2.4.0.0/schild-report.schild-report.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: schild-report.schild-report
+PackageVersion: 2.4.0.0
+PackageLocale: en-US
+Publisher: schild-report
+PackageName: schild.report
+License: MIT
+ShortDescription: schild.report ist eine moderne Lösung zum Erstellen von Reports mit Daten aus der Schild-Datenbank, die den bei Schild mitgelieferten Report-Designer vollständig ersetzen kann.
```
Comment (quhxl) [inline manifests/s/schild-report/schild-report/2.4.0.0/schild-report.schild-report.locale.en-US.yaml:10]:
```suggestion
ShortDescription: schild.report is a modern solution for creating reports with data from the Schild database, which can completely replace the report designer supplied with Schild.
```

File: manifests/s/schild-report/schild-report/2.4.0.0/schild-report.schild-report.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: schild-report.schild-report
+PackageVersion: 2.4.0.0
+PackageLocale: en-US
+Publisher: schild-report
```
Comment (quhxl) [inline manifests/s/schild-report/schild-report/2.4.0.0/schild-report.schild-report.locale.en-US.yaml:7]:
```suggestion
Publisher: '${env.THE_AUTHOR}'
PublisherSupportUrl: https://github.com/schild-report/schild.report/issues
```

File: manifests/s/schild-report/schild-report/2.4.0.0/schild-report.schild-report.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: schild-report.schild-report
+PackageVersion: 2.4.0.0
+PackageLocale: en-US
+Publisher: schild-report
+PackageName: schild.report
+License: MIT
```
Comment (quhxl) [inline manifests/s/schild-report/schild-report/2.4.0.0/schild-report.schild-report.locale.en-US.yaml:9]:
```suggestion
License: MIT
LicenseUrl: https://github.com/schild-report/schild.report/blob/master/LICENSE
Copyright: Copyright (c) hmt
```

File: manifests/s/schild-report/schild-report/2.4.0.0/schild-report.schild-report.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: schild-report.schild-report
+PackageVersion: 2.4.0.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
```
Comment (quhxl) [inline manifests/s/schild-report/schild-report/2.4.0.0/schild-report.schild-report.installer.yaml:9]:
```suggestion
  InstallerType: msi
  Scope: user
```

File: manifests/s/schild-report/schild-report/2.4.0.0/schild-report.schild-report.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: schild-report.schild-report
+PackageVersion: 2.4.0.0
```
Comment (quhxl) [inline manifests/s/schild-report/schild-report/2.4.0.0/schild-report.schild-report.installer.yaml:5]:
```suggestion
PackageVersion: 2.4.0.0
InstallModes:
- silent
- silentWithProgress
```

File: manifests/s/schild-report/schild-report/2.4.0.0/schild-report.schild-report.locale.en-US.yaml (hunk 2 above)
Comment (quhxl) [reply manifests/s/schild-report/schild-report/2.4.0.0/schild-report.schild-report.locale.en-US.yaml:7]: grafik

**Review by quhxl — CHANGES_REQUESTED** [review]
Please try to add as much metadata as possible in the future.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**